### PR TITLE
Add some stubs and missing symbols for CoreGraphics

### PR DESCRIPTION
**Pull request by dingari**

### CODE REVIEW

File: CoreGraphics/include/CoreGraphics/CGWindowLevel.h [hunk 1]
```diff
@@ -14,4 +14,37 @@ enum {
...
+    kCGScreenSaverWindowLevelKey,
+    kCGMaximumWindowLevelKey,
+    kCGOverlayWindowLevelKey,
+    kCGHelpWindowLevelKey,
+    kCGUtilityWindowLevelKey,
+    kCGDesktopIconWindowLevelKey,
+    kCGCursorWindowLevelKey,
+    kCGAssistiveTechHighWindowLevelKey,
+    kCGNumberOfWindowLevelKeys
+};
```
Comment (CuriousTommy) [inline CoreGraphics/include/CoreGraphics/CGWindowLevel.h:42]: I already talked about this in the other PR, but i recommend sticking with how [Apple does in their documentation](https://developer.apple.com/documentation/coregraphics/cgwindowlevelkey?language=objc), or (even better) use `NS_ENUM`.

File: CoreGraphics/include/CoreGraphics/CGGradient.h [hunk 2]
```diff
@@ -9,6 +9,8 @@ enum {
     kCGGradientDrawsAfterEndLocation = 0x02
 };
 
+typedef uint32_t CGGradientDrawingOptions;
```
Comment (CuriousTommy) [inline CoreGraphics/include/CoreGraphics/CGGradient.h:12]: Add in [kCGGradientDrawsBeforeStartLocation](https://developer.apple.com/documentation/coregraphics/cggradientdrawingoptions/kcggradientdrawsbeforestartlocation?language=objc) and [kCGGradientDrawsAfterEndLocation](https://developer.apple.com/documentation/coregraphics/cggradientdrawingoptions/kcggradientdrawsafterendlocation?language=objc).

https://developer.apple.com/documentation/coregraphics/cggradientdrawingoptions?language=objc

**Review by CuriousTommy — CHANGES_REQUESTED** [review]

File: CoreGraphics/include/CoreGraphics/CGWindowLevel.h (hunk 1 above)
Comment (dingari) [reply CoreGraphics/include/CoreGraphics/CGWindowLevel.h]: Using NS_ENUM resulted in a lot of compilation errors, but I've replaced these with CF_ENUM instead.

File: CoreGraphics/include/CoreGraphics/CGGradient.h (hunk 2 above)
Comment (dingari) [reply CoreGraphics/include/CoreGraphics/CGGradient.h]: I'm a bit confused, these constants were already defined, but I just added the typedef.

Anyway,  I've updated this enum declaration to use CF_ENUM, so it should be fine.

Comment (CuriousTommy) [reply CoreGraphics/include/CoreGraphics/CGGradient.h]: > I'm a bit confused, these constants were already defined, but I just added the typedef.

Ah! My bad, I didn't see that those were already there.

File: CoreGraphics/include/CoreGraphics/CGWindowLevel.h (hunk 1 above)
Comment (CuriousTommy) [reply CoreGraphics/include/CoreGraphics/CGWindowLevel.h]: > I've replaced these with CF_ENUM instead.

That's fine with me.

**Pull request merged**